### PR TITLE
chore: upgrade to QT 5.15.2

### PR DESCRIPTION
**Pull request by richard-ramos**
replaces https://github.com/status-im/status-desktop/pull/2712

### CODE REVIEW

**Review by pinebit — DISMISSED** [review]

**Review by jakubgs — APPROVED** [review]
Not sure why the new PR was created...

File: scripts/windows_build_setup.ps1 [hunk 1]
```diff
@@ -74,7 +74,7 @@ export PATH=`"/c/ProgramData/scoop/apps/inno-setup/current:`$PATH`"
 # Stop the script after first error
 $ErrorActionPreference = 'Stop'
 # Version of Qt SDK available form aqt
-$QtVersion = "5.14.2"
+$QtVersion = "5.15.2"
```
Comment (jakubgs) [inline scripts/windows_build_setup.ps1:77]: Have you tested if this works?

Comment (richard-ramos) [reply scripts/windows_build_setup.ps1:77]: I have not

**Comment by iurimatias** [comment]
closed due to upcoming c++ app that uses QT6

**Pull request closed without merging**